packages/plugins/rubygem-foreman-tasks/rubygem-foreman-tasks.spec

```diff
@@ -117,6 +121,7 @@ exit 0
 %{foreman_apipie_cache_foreman}
 %{foreman_apipie_cache_plugin}
 %{foreman_dir}/script/foreman-debug.d/60-dynflow_debug
+%config(noreplace) %{_sysconfdir}/logrotate.d/%{gem_name}
```

Comment:
Question, looks like the file ended up in:

```
# rpm -ql tfm-rubygem-foreman-tasks | grep logro
/opt/theforeman/tfm/root/etc/logrotate.d/foreman-tasks
```

How is this possible? I see not SCL prefix in the install and file section. Any hint @ekohl ?

Reply:
When the rpm is built inside the SCL the rpm macros point at paths inside the SCL too.

Reply:
You're looking for `_root_sysconfdir` so it's actually used.